### PR TITLE
CN-5104: reworked celpay flow

### PR DESCRIPTION
**Pull request by srkela**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [ ] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/layouts/RegularLayout/RegularLayout.js [hunk 1]
```diff
@@ -33,16 +33,37 @@ class RegularLayout extends Component {
     fabType: PropTypes.oneOf(FAB_TYPE),
     refreshing: PropTypes.bool,
     pullToRefresh: PropTypes.func,
+    getTransactionDetails: PropTypes.func,
```
Comment (fjovakaric) [inline app/components/layouts/RegularLayout/RegularLayout.js:36]: ?

Comment (fjovakaric) [reply app/components/layouts/RegularLayout/RegularLayout.js:36]: Remoove refreshing

File: app/components/screens/CelPayLanding/CelPayLanding.js [hunk 2]
```diff
@@ -77,7 +54,9 @@ class CelPayLanding extends Component {
   sendToFriend = () => {
     const { actions } = this.props;
 
-    actions.navigateTo("CelPayChooseFriend");
+    actions.navigateTo("CelPayEnterAmount", {
```
Comment (fjovakaric) [inline app/components/screens/CelPayLanding/CelPayLanding.js:57]: Use `CEL_PAY_TYPES`

File: app/components/screens/CelPayLanding/CelPayLanding.js [hunk 3]
```diff
@@ -133,7 +110,7 @@ class CelPayLanding extends Component {
       );
 
     return (
-      <RegularLayout refreshing={refreshing} pullToRefresh={this.refresh}>
+      <RegularLayout pullToRefresh={() => actions.getAllTransactions()}>
```
Comment (fjovakaric) [inline app/components/screens/CelPayLanding/CelPayLanding.js:113]: only celpay transactions
